.github/actions/windows-signing/config_x64.toml

```diff
@@ -52,5 +44,4 @@ name = "OBS-Studio-{version}-x64.zip"
 pdb_name = "OBS-Studio-{version}-pdbs.zip"
 
 [package.updater]
-vc_redist_path = "bouf/nsis/VC_redist.x64.exe"
```

Comment:
Is this no longer needed?

Reply:
Yeah it's only for the update manifest, which this job does not create.